tf-psa-crypto/core/psa_crypto_core.h

```diff
@@ -155,7 +155,11 @@ typedef struct {
     /* Dynamically allocated key data buffer.
      * Format as specified in psa_export_key(). */
     struct key_data {
+#if defined(MBEDTLS_PSA_STATIC_KEY_SLOTS)
+        uint8_t data[MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE];
+#else /* MBEDTLS_PSA_STATIC_KEY_SLOTS */
         uint8_t *data;
+#endif /* MBEDTLS_PSA_STATIC_KEY_SLOTS */
```

Comment:
Since the span of this `#if-else-endif` is so short, it's probably better for readability to remove the `MBEDTLS_PSA_STATIC_KEY_SLOTS` comments on the `#else` and `#endif` lines